android/src/main/java/com/urbanairship/reactnative/UrbanAirshipReactModule.java

```diff
@@ -98,6 +99,17 @@ public void onHostPause() {
             public void onHostDestroy() {
 
             }
+
+            public void checkOptIn(Context context) {
+                boolean optIn = UAirship.shared().getPushManager().isOptIn();
```

Comment:
is there any way to avoid repeating ourselves here? it looks like a 1:1 copy of the checkOptIn above